src/echo/ws-api/wallet-api.js

```diff
@@ -382,55 +349,45 @@ class WalletAPI {
 	 * @returns {Promise<[string, string][]>}
 	 */
 	listAccounts(accountName, limit = API_CONFIG.LIST_ACCOUNTS_DEFAULT_LIMIT) {
-		if (!isString(accountName)) throw new Error('account name should be a string');
-		if (!isUInt32(limit) || limit > API_CONFIG.LIST_ACCOUNTS_MAX_LIMIT) {
-			throw new Error(`Limit should be a integer and must not exceed ${API_CONFIG.LOOKUP_ACCOUNTS_MAX_LIMIT}`);
+		if (!limit > API_CONFIG.LIST_ACCOUNTS_MAX_LIMIT) {
```

Comment:
`!limit` is always equals to `false` or `true`. So this field will be valid in any way.
Remove `!` here and below.